WinUIGallery/ControlPages/InfoBadgePage.xaml

```diff
@@ -86,9 +88,11 @@
 
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
-&lt;InfoBadge x:Name="infoBadge2" Style="{StaticResource $(Style)IconInfoBadgeStyle}" /&gt;
+&lt;InfoBadge x:Name="infoBadge2" Style="{StaticResource $(Style)IconInfoBadgeStyle}"
```

Comment:
The xaml here does not match the xaml above (and it didn't before the change :)).